src/Logary/Configuration_FactoryApi.fs

```diff
@@ -162,5 +214,5 @@ type LogaryFactory =
     if serviceName = null then nullArg "serviceName"
     if configurator = null then nullArg "configurator"
     let config = Config.confLogary serviceName
-    let cb = configurator.Invoke (ConfBuilder config)
-    cb.BuildLogary () |> CSharp.toTask
\ No newline at end of file
+    let cb = configurator.Invoke (ConfBuilder (config,LogLevel.Fatal) )
```

Comment:
it is the default internal logging level , use when people not setting it from `InternalLoggingLevel` . see comments after.